### ISSUE TITLE
DB/Creature: Fix Buzzard Movement

### ISSUE THREAD

**Issue by z3k-org** [TrinityCore/TrinityCore#30497]
### Description

Badlands Buzzard is flying in the ground.

### SQL fix

```sql
-- Badlands Buzzard
DELETE FROM `creature_template_movement` WHERE `CreatureId`=2830;
```

### Branch

3.3.5

### TC rev. hash/commit

https://github.com/TrinityCore/TrinityCore/commit/c354f25e5dec1ead1f12118aa42d51c91c60d6e9

**Comment by CraftedRO** [comment]
this is not how a fix shoud look like, we don't delete the entire row but we update the values with sniffed ones.

**Comment by CraftedRO** [comment]
looks like creature_template_movement is correct as it is according to sniff but the trick is in creature_template_addon > AnimTier, where wpp export value 3 and in sniff we read PlayHoverAnim: True, so even if we choose 2 for Buzzard it still looks weird ingame anyway probably because MovementType =1. most likely there's already an opened issue for this or at least a mention.

**Comment by CraftedRO** [comment]
ah, here it is https://github.com/TrinityCore/TrinityCore/issues/27366

**Comment by z3k-org** [comment]
Sorry my bad, I didnt know that.

I got no DB errors, the movement seemed fine and everything worked. So I decided to contribute a bit. Even tho its just a visual.

**Comment by CraftedRO** [comment]
Yeah, every moving flying npc spawned on ground (Z) level that have record in template movement are is in the same situation.

**Comment by offl** [comment]
> Sniffed fix still better than what we have atm:
> 
> -- Buzzard
> UPDATE `creature_template_addon` SET `AnimTier`=3,`PvPFlags`=0 WHERE `entry`=2830;

But these values are already in DB

**Comment by offl** [comment]
I really doubt they should fly, check creature difficulty flags of that creature
This flag https://github.com/TrinityCore/TrinityCore/blob/150d6995bdc966a5118e65d9bf6b6171faf9c62c/src/server/game/Entities/Creature/CreatureData.h#L67

**Comment by CraftedRO** [comment]
>But these values are already in DB

maybe because was added in the commit above ?!

idk but I'm repeating myself: every moving flying npc spawned on ground (Z) level that have record in template movement is in the same situation (will go through texture/ground), by removing creature_template_movement completely they behave blizzlike.

**Comment by offl** [comment]
I see now

**Comment by CraftedRO** [comment]
yeah, and indeed the sniffed values were taken from retail npc that have different name Parched Buzzard but same entry (not sure if smth change related to AnimTier state since then), iirc there are some gargoyles near Dragonblight-Wintergarde Keep in the same situation and many more...

**Comment by offl** [comment]
So the fix is
```sql
--
UPDATE `creature_template_movement` SET `Ground` = 1, `Flight` = 0 WHERE `CreatureId` = 2830;
```

**Comment by CraftedRO** [comment]
> So the fix is

I think we can merge this, looks good ingame.
